packages/twenty-server/src/integrations/message-queue/interfaces/message-queue-job.interface.ts

```diff
@@ -1,4 +1,4 @@
-export interface MessageQueueJob<T extends MessageQueueJobData> {
+export interface MessageQueueJob<T extends MessageQueueJobData | undefined> {
```

Comment:
I don't see a case where we want the payload to be undefined? Why did we need to add this?